documentation/asciidoc/topics/proc_creating_grafana_datasources.adoc

```diff
@@ -0,0 +1,65 @@
+[id='creating-grafana-dashboards_{context}']
+= Creating Grafana Data Sources
+Set up Prometheus as a data source so you can visualize {brandname} metrics in Grafana dashboards.
+
+.Prerequisites
+
+* Create a Prometheus service monitor for your {brandname} cluster.
+* Install the Grafana operator in your {brandname} cluster namespace.
```

Comment:
The grafana operator doesn't have to be in the same namespace as the cluster. It's possible to have a global grafana operator that consumes the `GrafanaDashboard` installed in any namespace. The namespace where the `GrafanaDashboard` CR is created is configured via the [`infinispan-operator-config` configmap](https://github.com/infinispan/infinispan-operator/blob/master/deploy/cr/operator-config/operator-config.yaml) and the `grafana.dashboard.namespace` key.